assets/scss/components/_ie-fixes.scss

```diff
@@ -0,0 +1,95 @@
+html.ie9 {
+    .Header {
+        height: 50px;
+
+        > * {
+            float: left;
+        }
+    }
+
+    .Frame-handle {
+        display: none;
+    }
+
+    .Frame-body {
+        height: auto;
+        width: 100%;
+    }
+    .Frame-inner {
+        margin-top: 50px;
+    }
+    .Frame-panel {
+        overflow: auto;
+    }
+    .Frame-panel--sidebar {
```

Comment:
@allmarkedup @tlenex What's the status on this, are we waiting to hear if the changes had been made to make it a sidebar?